TeslaMateAgile.Tests/IntegrationTests.cs

```diff
@@ -78,4 +78,46 @@ public async Task IntegrationTests_Awattar()
         Assert.LessOrEqual(priceData.Min(x => x.ValidFrom), from);
         Assert.GreaterOrEqual(priceData.Max(x => x.ValidTo), to);
     }
+
+    [Ignore(IntegrationTest)]
+    [Test]
+    public async Task IntegrationTests_Energinet()
+    {
+        var configBuilder = new ConfigurationBuilder()
+            .AddInMemoryCollection(new Dictionary<string, string>
+            {
+                ["Energinet:BaseUrl"] = "https://api.energidataservice.dk/dataset/",
+                ["Energinet:Region"] = "DK1",
+                ["Energinet:VAT"] = 1.25
```

Comment:
This `1.25` will need to be a string as it's pretending to be an environment variable as part of this test